python/ray/tests/test_node_labels.py

```diff
@@ -27,6 +27,38 @@ def test_ray_start_set_empty_node_labels(call_ray_start):
     assert ray.nodes()[0]["Labels"] == {}
 
 
+def test_ray_init_set_node_labels(shutdown_only):
+    labels = {"gpu_type": "A100", "region": "us"}
+    ray.init(labels=labels)
+    assert ray.nodes()[0]["Labels"] == labels
+    ray.shutdown()
+    ray.init(labels={})
+    assert ray.nodes()[0]["Labels"] == {}
+
+
+def test_ray_init_set_node_labels_value_error(ray_start_cluster):
+    cluster = ray_start_cluster
+    cluster.add_node(num_cpus=1)
+    with pytest.raises(ValueError, match="labels must not be provided"):
+        ray.init(address=cluster.address, labels={"gpu_type": "A100"})
```

Comment:
If enable the node label feature, should all the node must add label?

Reply:
labels are optional for nodes.